public/api/conf/1.0/examples/national-insurance-record-example-1.json

```diff
@@ -134,7 +134,7 @@
         "classThreeCredits": 0,
         "otherCredits": 52,
         "classThreePayable": 0,
-        "classThreePayableBy": null,
+        "classThreePayableBy": "2019-04-05",
```

Comment:
classThreePayable should not be included if the the year is qualifying.

classThreePayable and classThreePayableByPenalty should be there if the year is a gap.